### PR TITLE
Landing Page and styled Filtering Page

### PR DESCRIPTION
**Pull request by canhuynh1998**

### CODE REVIEW

**Comment by canhuynh1998** [comment]
Need to add user authentication so the navbar can show properly. It currently shows logged in view

File: views/templates/partials/nav.ejs [hunk 1]
```diff
@@ -0,0 +1,15 @@
+<nav class="text-gray-700 bg-gray-400 body-font m">
+    <ul class="container mx-auto flex flex-row-reverse p-10 md:flex-row-reverse ">
+        <%if (isLoggedIn) { 
+            %>
+            <li class="mx-2"><a href="#">Log Out</a></li>
+            <li class="mx-2"><a href="#">Profile</a></li>
+            <li class="mx-2"><a href="#">Artist</a></li>
+            <li class="mx-2"><a href="/concert-search">Concert</a></li>
+
```
Comment (euniceyoh) [inline views/templates/partials/nav.ejs:10]: do u think we should add a tab to also get back to the home / landing page?

**Review by euniceyoh — COMMENTED** [review]
looks really nice!

File: views/templates/partials/nav.ejs (hunk 1 above)
Comment (canhuynh1998) [reply views/templates/partials/nav.ejs:10]: yeah, I can do it. I think that tap should be accessible to everyone like authenticated or not can see that tap

Comment (euniceyoh) [reply views/templates/partials/nav.ejs:10]: yeah that makes sense

**Pull request merged**